docs/modules/fs-extra.md

```diff
@@ -0,0 +1,74 @@
+| `move(src, dest, { overwrite: true }, callback)`                                                              | `fs.rename(source, destination)`                          |                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| `moveSync(source, destination, { overwrite: true })`                                                          | `fs.renameSync(source, destination)`                      | These are not strictly equivalent. The `move` methods in fs-extra act like the `mv` command and they will work across devices also. The `rename` methods in fs work like the `rename` system call and will **not** work across devices. Having Node’s `rename` methods work across devices is currently [marked as a "won't fix."](https://github.com/nodejs/node/issues/19077) Also, the `overwrite: true` behaviour is the default/only behaviour in the standard library. |
+| `copy(source, destination, callback)`                                                                         | `fs.cp(source, destination, callback)`                    |                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| `pathExistsSync(file)`                                                                                        | `fs.existsSync(path)`                                     |                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+
+#### Methods that need migrating or more than one `node:fs` method
+
+##### `emptyDir(dir[, callback])`
```

Comment:
for all of these, I would just mention something to summarise instead:

> For all other methods, a combination of `fs` methods may be needed rather than there being a one-to-one replacement. See the docs [here](#)